lms/djangoapps/grades/new/course_grade.py

```diff
@@ -72,10 +66,7 @@ def grade_value(self):
             self.subsection_grade_totals_by_format,
             generate_random_scores=settings.GENERATE_PROFILE_SCORES
         )
-        log.info(u"Persistent Grades: Calculated grade_value. course id: {0}, user: {1}".format(
-            self.course.location,
-            self.student.id
-        ))
+        self._log_event(log.info, u"grade_value")
```

Comment:
Same comment as elsewhere- I think that when we are passing a straight-up string, the `u` is unnecessary.